packages/compiler/src/server/tspconfig/completion.ts

```diff
@@ -80,6 +111,110 @@ export async function provideTspconfigCompletionItems(
         itemsFromEmitter.push(...more);
       }
       return [...itemsFromBuiltIn, ...itemsFromEmitter];
+    } else if (nodePath.length > CONFIG_PATH_LENGTH_FOR_EMITTER_LIST && nodePath[0] === "linter") {
```

Comment:
please add a CONFIG_PATH... for linter. the CONFIG_PATH_LENGTH_FOR_EMITTER_LIST has nothing to do with linter.

Reply:
same comment for followings